addons/stock/models/stock_lot.py

```diff
@@ -177,6 +178,12 @@ def _set_single_location(self):
         elif len(quants.location_id) > 1:
             raise UserError(_('You can only move a lot/serial to a new location if it exists in a single location.'))
 
+    def _compute_is_scrap(self):
```

Comment:
It's stored without depends. There is so many way it remains never scrap despite new moves. I'm not sure we should store is_scrap on lot. Or if we do it should be available in the view